<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Much threads, such MVC, so semaphore</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="12" column="5">
     <widget class="QPushButton" name="pushButton_pelemele">
      <property name="text">
       <string>Pêle-mêle</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_selectImages">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>27</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;/home/thierry/Images/ressource_image.jpeg&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="3">
     <widget class="QPushButton" name="pushButton_setZValue">
      <property name="text">
       <string>Premier plan</string>
      </property>
     </widget>
    </item>
    <item row="12" column="6">
     <widget class="QSpinBox" name="spinBox">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Taille du filtre&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="12" column="9">
     <widget class="QPushButton" name="pushButton_ouvrirVideo">
      <property name="text">
       <string>Ajouter Vidéo</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" rowspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <widget class="QTextEdit" name="textEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>36</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 199, 199);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Open Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Liste&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Transformer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="10" rowspan="10">
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_selectVideos">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>27</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;/home/thierry/Images/ressource_movie.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="12" column="8">
     <widget class="QPushButton" name="pushButton_sequencage">
      <property name="text">
       <string>Séquençage</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="pushButton_ouvrirImage">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ajouter Image</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="10" colspan="9">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>523</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="10">
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 199, 199);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Open Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Fenêtre de travail&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="ouvrirImage"/>
    <addaction name="ouvrirVideo"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>Images</string>
    </property>
    <addaction name="pelemeleNB"/>
    <addaction name="pelemeleCouleur"/>
    <addaction name="pelemeleGauss"/>
   </widget>
   <widget class="QMenu" name="menuVid_os">
    <property name="title">
     <string>Vidéos</string>
    </property>
    <addaction name="seqX"/>
    <addaction name="seqY"/>
   </widget>
   <widget class="QMenu" name="menuTravailler_avec">
    <property name="title">
     <string>Travailler avec :</string>
    </property>
    <addaction name="focusImage"/>
    <addaction name="focusVideo"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuTravailler_avec"/>
   <addaction name="menuImages"/>
   <addaction name="menuVid_os"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ouvrirImage">
   <property name="text">
    <string>Ouvrir un fichier image</string>
   </property>
  </action>
  <action name="pelemeleNB">
   <property name="text">
    <string>Pêle-mêle noir &amp; blanc</string>
   </property>
  </action>
  <action name="pelemeleCouleur">
   <property name="text">
    <string>Pêle-mêle couleurs</string>
   </property>
  </action>
  <action name="seqX">
   <property name="text">
    <string>Séquençage horizontal</string>
   </property>
  </action>
  <action name="seqY">
   <property name="text">
    <string>Séquençage vertical</string>
   </property>
  </action>
  <action name="ouvrirVideo">
   <property name="text">
    <string>Ouvrir un fichier vidéo</string>
   </property>
  </action>
  <action name="focusImage">
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="focusVideo">
   <property name="text">
    <string>Vidéos</string>
   </property>
  </action>
  <action name="pelemeleGauss">
   <property name="text">
    <string>Pêle-mêle avec flou gaussien</string>
   </property>
  </action>
  <action name="detection_de_contour">
   <property name="text">
    <string>Pêle-mêle avec détection de contour</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>pushButton_pelemele</tabstop>
  <tabstop>pushButton_sequencage</tabstop>
  <tabstop>pushButton_ouvrirVideo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
